docs/contribute/where-to-work.md

```diff
@@ -0,0 +1,99 @@
+
+# **How do I know where to contribute?**
+
+The easiest way to find the source file for the page you want to update is to navigate to the page on the web and click **Edit this page**. This will take you to the source file in the correct repo.
+
+The following outlines the general guidelines though to help you determine where the content might be living.
+
+## **9.0+ content**
+
+For all 9.0+ content, it lives in one of three spots:
+
+1. **Reference/settings content** lives in the new Markdown files in the code repo as it did pre 9.0. This enables folks to make updates that touch code \+ low-level docs in the same PR.  
+2. **Narrative/conceptual and overview content** lives in [elastic/docs-content](https://github.com/elastic/docs-content). As an example the [ES|QL overview](https://docs-v3-preview.elastic.dev/elastic/docs-content/tree/main/explore-analyze/query-filter/languages/esql) lives in the **Explore & Analyze** section of the narrative docs. But the reference documentation lives in the reference section of the docs [ES|QL reference](https://docs-v3-preview.elastic.dev/elastic/elasticsearch/tree/main/reference/query-languages/esql)  
+3. **API reference docs** live in the different OpenAPI spec repositories. This is where you need to update API docs published in the new API docs system
```

Comment:
> **API reference docs** live in the different OpenAPI spec repositories. This is where you need to update API docs published in the new API docs system

If this content is going to persist as a public-facing document that we point all external and internal contributors to, I'm happy to beef up the "how to contribute to API docs" content here too.